packages/lsp/test/connection_manager.spec.ts

```diff
@@ -99,5 +100,12 @@ describe('@jupyterlab/lsp', () => {
         expect(manager.disconnectDocumentSignals).toHaveBeenCalled();
       });
     });
+    describe('#onUninitializedLSPConnectionDisposed', () => {
```

Comment:
We are describing the behaviour of the `dispose()` method here:

```suggestion
    describe('#dispose()', () => {
```